KenBurns/Classes/KenBurns.swift

```diff
@@ -195,6 +214,53 @@ func ==(lhs: KenBurnsAnimation, rhs: KenBurnsAnimation) -> Bool {
         updatesDisplayLink.add(to: RunLoop.main, forMode: .common)
         startNewAnimation()
     }
+    
+    public func setImageQueue(withImages images:[UIImage]) {
+        if images.count == 0 {
+            fatalError("This is an empty queue!")
+        }
+        
+        remoteQueue = false
+        index = 0
+        self.imageQueue = RingBuffer(count: images.count)
+        
+        for i in images {
+            self.imageQueue?.write(i)
+        }
+    }
+    
+    public func setImageQueue(withUrls urls:[URL], placeholders: [UIImage]?) {
```

Comment:
Good interface, no exposure of Kingfisher types or objects.

Reply:
🙏🏼Thank you